torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -124,7 +124,7 @@ void PropagateShapeOnNode(Node * node) {
   // XXX: real attributes of node can be a superset of attrs
   // XXX: if this returns true then you are obliged to set the types
   auto check_overload = [&](size_t num_inputs, size_t num_outputs,
-                            std::vector<std::pair<AttributeKind,Symbol>> attrs = {}) -> bool {
+                            std::vector<std::pair<AttributeKind,Symbol>> attrs){
```

Comment:
But why?

Reply:
I'll undo it now that I got rid of -pedantic